ruby_activerecord/app/models/feedback.rb

```diff
@@ -0,0 +1,7 @@
+class Feedback < ActiveRecord::Base
+  belongs_to :product
+
+  validates :rate, inclusion: 1..5, presence: true
+  validates :comment, length: { maximum: 256 }, presence: true
+  validates :username, length: { in: 3..56 }, presence: true
```

Comment:
presence validation will not work with default value in DB, when you put empty username. Because this validation will be called BEFORE default setting